Proposals/xgov-178.md

```diff
@@ -0,0 +1,87 @@
+- **NextJS Support**: SIWA will support NextJS, a popular React framework, making it easy for developers to build fast and scalable web applications.
+- **Enterprise Security**: SIWA will provide a best-practices security model and on-chain protocol for enterprise applications, ensuring that user data is secure and protected.
+
+Feature 1: EVM Compatibility
+SIWA will be modeled from the ground up to be compatible with SIWE, allowing developers to easily integrate SIWA into their existing Ethereum-based applications. This is a critical feature for developers who build Algorand applications with NextJS. This is a critical feature for developers who build EVM-compatible applications and want to leverage the Algorand blockchain for authentication.
+
+Feature 2: EIP-4361 Equivalent
+SIWA will implement an EIP-4361 equivalent on the Algorand blockchain. EI-4361 is an Ethereum Improvement Proposal that defines a standard for decentralized authentication on the Ethereum blockchain. SIWA will provide a similar standard for Algorand, allowing developers to easily integrate decentralized authentication into their applications.
+
+Feature 3: NextJS Support
+SIWA will support NextJS, a popular React framework, making it easy for developers to build fast and scalable web applications. NextJS is a powerful tool for building web applications, and SIWA will provide seamless integration with NextJS, allowing developers to build secure and scalable applications with ease.
+
+Feature 4: Wallet/NFD Integration
+SIWA will integrate with popular Algorand wallets, popular Algorand libraries, and NFD domains. This integration will allow users to easily sign in to websites and applications using their Algorand wallet, providing a seamless and secure authentication experience.
+
+## Team
+
+- Aaron Martinez, Lead Developer
+
+  - Role: Solo project lead
+  - Expertise: NextJS, React, Authentication
+  - Responsibilities: Project management, development, testing, deployment
+  - LinkedIn: https://www.linkedin.com/in/ussaaron
+
+## Experience with Algorand
+
+Aaron has been working with Algorand since 2021. He has recieved multiple development awards for the Pipeline-UI developer toolkit. He has extensive experience with the Algorand blockchain, including:
+
+- Pipeline-UI developer toolkit
+  - https://github.com/headline-design/pipeline
+
+- Algo-OS Open Source Campaign
+  - https://twitter.com/headline_crypto/status/1692964932944290294
+
+- XBallot Governance Protocol
+  - https://xballot.net
+
+- And much more!
+
+## Present Proposal
+
+Our grant application targets one critical milestones:
+
+1. **SIWA Protocol publication**: Protocol publication will include full documentation site, NPM package publication, and all relevent integrations for use in production.
+
+Cost Breakdown
+
+Protocol publication - total: (150,000A)
+
+- Full documentation site
+- NPM package publication
+- Production integrations
+
```

Comment:
Please provide public URL of where/how the community can check proposal deliverables.

Reply:
Absolute. Are you sating to add the url on line 75? or where exactly? I don't want to put it in the wrong place.

Reply:
The work on SIWA is currently being done here - https://github.com/headline-design/xballot/tree/main/packages/siwa. As soon as I have a working prototype I'll create a seperate repository for SIWA.

Reply:
Thrilled to announce that we now have a working prototype. Will share the Github repos, etc. in the coming days.